sql/_05_plcsql/_01_testspec/_04_expression/_04_function_call/_03_builtin_function/_05_datetime_function/cases/04_04_03_05_bfn_datetime_adddate.sql

```diff
@@ -8,7 +8,6 @@ begin
     dbms_output.put_line(ADDDATE(TO_DATETIME('1999-01-08'), NULL));
 
     -- ADDDATE() INTERVAL delimiter '' and option parse error
-    /*
     dbms_output.put_line(ADDDATE(NULL, INTERVAL 123 MILLISECOND));
```

Comment:
Currently, INTERVAL in ADDDATE, DATE_SUB, SUBDATE, etc  is not supported.
This and other lines that uses it should be erased.